src/Microsoft.Identity.Client/Core/Constants.cs

```diff
@@ -65,7 +65,7 @@ public static string FormatAdfsWebFingerUrl(string host, string resource)
         {
             return string.Format(
                 CultureInfo.InvariantCulture,
-                "https://{0}/adfs/.well-known/webfinger?rel={1}&resource={2}",
```

Comment:
>adfs/ [](start = 29, length = 5)

is this used in ADFS 2016?

Reply:
FormatAdfsWebFingerUrl() was added to the constants class to properly format this url

Reply:
sure, however my question is around ADFS 2016. Is this formatted with adfs/ in ADFS 2016?

---
In reply to: [255647716](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/834#discussion_r255647716) [](ancestors = 255647716)

Reply:
@reed1995 ?

---
In reply to: [255758669](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/834#discussion_r255758669) [](ancestors = 255758669,255647716)